core/frontend/src/components/parameter-editor/ParameterLoader.vue

```diff
@@ -270,12 +288,22 @@ export default Vue.extend({
         this.$set(this.param_checkboxes, name, true)
       }
     },
-    createPrintableParams(paramset: Dictionary<number>) {
-      return Object.entries(paramset).map(([name, value]) => ({
-        name,
-        value,
-        current_value: autopilot_data.parameter(name)?.value,
-      }))
+    parametersFromSet(paramset: Dictionary<number>) {
+      return Object.entries(paramset).map(([name, value]) => {
+        const currentParameter = autopilot_data.parameter(name)
+
+        return {
+          name,
+          current: currentParameter,
+          new: { ...currentParameter, value },
+        }
+      })
+    },
+    prettyNameFromParameter(parameter: Parameter) {
+      const paramValueText = printParam(parameter)
+      const paramUnitsText = parameter?.units ? `[${parameter.units}]` : ''
+
+      return paramValueText + paramUnitsText
```

Comment:
```suggestion
      return paramValueText + ' ' + paramUnitsText
```

I don't remember the correct syntax, but the idea is to have something like "10 [A]" instead of "10A".